lib/inspec/dependencies/requirement.rb

```diff
@@ -10,7 +10,18 @@ module Inspec
   class Requirement
     def self.from_metadata(dep, cache, opts)
       raise 'Cannot load empty dependency.' if dep.nil? || dep.empty?
-      new(dep[:name], dep[:version], cache, opts[:cwd], opts.merge(dep))
+
+      req_path = opts[:cwd]
+
+      if dep[:path]
+        req_path = File.expand_path(dep[:path], req_path)
+      end
+
+      new(dep[:name],
+          dep[:version],
+          cache,
+          File.absolute_path(req_path),
```

Comment:
`File.expand_path` [here](https://github.com/chef/inspec/pull/2121/files#diff-c31bbcc4f73daa9c1b95020b03f5303dR17) should do the trick, right? We probably don't even need to call on `File.absolute_path` anymore.

Reply:
Heh, completely missed that. Updated.